terrautils/sensors.py

```diff
@@ -53,386 +12,374 @@
-                'template': '{TERRAREF_BASE}/{station}/{level}/'
+                'template': '{base}/{station}/{level}/'
                             '{sensor}/{date}/{filename}',
-                'pattern': '{}_{}_environmentlogger.nc'.\
-                           format(date_p, time_p)
+                'pattern': '{sensor}_{level}_{station}_{timestamp}{opts}.nc',
             },
+
             'soil_removal_vnir': {
-                'template': '{TERRAREF_BASE}/{station}/{level}/'
+                'template': '{base}/{station}/{level}/'
                             '{sensor}/{date}/{time}/{filename}',
-                'pattern': 'VNIR_lv1_{}__{}_{}'.format(date_p, 
-                           full_time_p, 'uamac_soilremovalmask.nc')
+                'pattern': 'VNIR_{level}_{station}_{timestamp}{opts}.nc'
             },
+
             'soil_removal_swir': {
-                'template': '{TERRAREF_BASE}/{station}/{level}/'
+                'template': '{base}/{station}/{level}/'
                             '{sensor}/{date}/{time}/{filename}',
-                'pattern': ''
+                'pattern': 'SWIR_{level}_{station}_{timestamp}{opts}.nc'
             },
+
             'scanner3DTop_mergedlas': {
-                'template': '{TERRAREF_BASE}/{station}/{level}/'
-                            '{sensor}/{date}/{time}/{filename}',
-                'pattern': 'scanner3DTop_(lv1_)?{}__{}.las'.\
-                           format(date_p, full_time_p,
-                           '(_uamac_merged | MergedPointCloud)')
+                'template': '{base}/{station}/{level}/'
+                            'scanner3DTop/{date}/{time}/{filename}',
+                'pattern': 'scanner3DTop_{level}_{station}_{timestamp}'
+                           '_merged{opts}.las'
             },
+
             'scanner3DTop_plant_height': {
-                'template': '{TERRAREF_BASE}/{station}/{level}/'
-                            '{sensor}/{date}/{time}/{filename}',
-                'pattern': ('scanner3DTop - {}__{} {}.npy'.\
-                            format(date_p, full_time_p,
-                              '(highest|historgram)'))
+                'template': '{base}/{station}/{level}/'
+                            'scanner3DTop/{date}/{time}/{filename}',
+                'pattern': 'scanner3DTop_{level}_{station}_{timestamp}'
+                           '_height{opts}.npy'
             },
+
             'scanner3DTop_heightmap': {
-                'template': '{TERRAREF_BASE}/{station}/{level}/'
+                'template': '{base}/{station}/{level}/'
                             '{sensor}/{date}/{time}/{filename}',
-                'pattern': ('scanner3DTop_lv1_{}__{}_{}.bmp'.\
-                            format(date_p, full_time_p,
-                            'uamac_heightmap(_mask)?'))
-            }
-        },
-        'raw_data': {
-            'stereoTop': {
-                "fixed_metadata_datasetid": "5873a8ae4f0cad7d8131ac0e",
```

Comment:
All these got removed somehow - can you do a pull from master on this branch? Craig added all these IDs intentionally.